src/status_im/contexts/wallet/events.cljs

```diff
@@ -327,10 +327,11 @@
  :wallet/set-ens-address
  (fn [{:keys [db]} [result ens]]
    (let [suggestion (if result
-                      [{:type     item-types/address
-                        :ens      ens
-                        :address  (eip55/address->checksum result)
-                        :networks [:ethereum :optimism]}]
+                      [{:type         item-types/address
+                        :ens          ens
+                        :address      (eip55/address->checksum result)
+                        :networks     [:ethereum :optimism]
+                        :full-address (str "eth:opt:" (eip55/address->checksum result))}]
```

Comment:
I see this `"eth:opt:" string as a magical string. It could be a constant or better yet IMO, a function that converts a sequence of network keywords into their shorthand versions. I can't remember, but I think such a function even exists already, but don't trust my memory.

Reply:
I wrote some functions to convert network ids to address prefix: 

```clj
(def network->short-name
  {constants/mainnet-network-name  constants/mainnet-short-name
   constants/optimism-network-name constants/optimism-short-name
   constants/arbitrum-network-name constants/arbitrum-short-name})

(def short-name->network
  {constants/mainnet-short-name  constants/mainnet-network-name
   constants/optimism-short-name constants/optimism-network-name
   constants/arbitrum-short-name constants/arbitrum-network-name})

(defn short-names->network-preference-prefix
  [short-names]
  (str (string/join ":" short-names) ":"))

(defn network-preference-prefix->network-names
  [prefix]
  (as-> prefix $
    (string/split $ ":")
    (map short-name->network $)))
 ```
These were in `wallet.common.utils` ns. But my PR was not merged because I was moved to another team and Ibrahim started working on that feature.

Agreed that we should have a namespace to manage addresses. They are not strings. They just look like strings.

Reply:
I'm gonna copy your code and put it in my PR @shivekkhurana

Reply:
`status-im.contexts.wallet.common.utils.networks` This will be a better namespace to include them in.